apps/aehyperchains/src/aehc_app.erl

```diff
@@ -33,9 +38,9 @@ stop(_State) ->
 check_env() ->
     case aehc_utils:hc_enabled() of
         true ->
-            lager:info("Hyperchains are enabled");
+            linfo("Hyperchains are enabled");
         false ->
-            lager:info("Hyperchains are disabled"),
+            linfo("Hyperchains are disabled"),
```

Comment:
I'm very fond of consistency in the code. Now I'm a bit confused – what should we use for logging from now on, `lager` or `aehc_log`?

Reply:
We should use lager

Reply:
> I'm very fond of consistency in the code. Now I'm a bit confused – what should we use for logging from now on, `lager` or `aehc_log`?

My opinion is we should abstract such things. Lager? logger? io:format? whatever?.. Those changes shouldn't be a concern for a higher level code. And abstracting is the only way to move them out of concern. Again, just my opinion here.